scripts/install_shiny_deps.R

```diff
@@ -0,0 +1,16 @@
+
+# Install dependencies declared by a Shiny app
+# Relies on the undocumented behavior that devtools::install_deps will work
+# on any directory containing a DESCRIPTION file, even if not an R package.
+
+path <- commandArgs(trailingOnly = TRUE)[[1]]
```

Comment:
No need to change, just curious: Why use double brackets here? Doesn't `commandArgs` always return a character vector?

Reply:
I think it was just habit here ("I'm sure I only want one value back, better use double brackets"), but now that I think about it `commandArgs()[[1]]` has the virtue of failing loudly if you forget to pass any arguments.